README.md

```diff
@@ -351,6 +351,8 @@ This annotation tells the check-checker to consider the specified variables to b
 
 ### Execute a test suite
 
+> Warning: `clarinet test` will soon be deprecated in favor of a new way of testing smart contracts. The new documentation is under construction. In the meantime, learn more [in the clarinet-sdk Readme](https://github.com/hirosystems/clarinet/blob/01da3550670f321a2f19fd3b0f8df0fb4b769b08/components/clarinet-sdk/README.md).
```

Comment:
@hugocaillard I'll leave it up to you, but I wonder if this should be bigger and at the top of the whole readme so users are more likely to see it

Reply:
@MicaiahReid Hmm yeah it would make sense. Also it'll be quite visible on docs.hiro.so.

For now, I think I'd like to keep it quite discrete until I have a clean documentation page to link to (EOW).

In this PR I really wanted to fix #1149 (which impact the `test` command so I added a small warning